tensorflow/compiler/xla/service/gpu/tests/BUILD

```diff
@@ -812,10 +812,6 @@ xla_cc_binary(
     ]) + if_rocm_is_configured([
         "//tensorflow/tsl/platform:rocm_rocdl_path",
         "//tensorflow/compiler/xla/service/gpu:amdgpu_compiler_impl",
-<<<<<<< HEAD
-	"//tensorflow/tsl/platform:rocm_rocdl_path",
```

Comment:
This was added in: https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/dfb7dd0b2f7e6c6bbcd3f1dbc9e299be9554b4a7

At the time, it was needed. Curious if that is still the case.